client/cmd/playground/window/model.go

```diff
@@ -50,35 +72,39 @@ func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msgStr {
 	case "ctrl+c", "q":
 		return m, tea.Quit
-	case "up", "w":
+	case "up":
 		m.handleUp()
-	case "down", "s":
+	case "down":
 		m.handleDown()
-	case "left", "a":
+	case "left":
 		m.handleLeft()
-	case "right", "d":
+	case "right":
 		m.handleRight()
 	case "shift+up", "W":
 		m.handleIncrement()
 	case "shift+down", "S":
 		m.handleDecrement()
-	case "M", "h", "-",
-		"1", "2", "3", "4", "5",
-		"6", "7", "8", "9", "0",
-		"backspace":
+	default:
 		m.handleInput(msg)
 	}
 	return m, nil
 }
 
 func (m *model) View() string {
+	buff := new(bytes.Buffer)
+	table := tablewriter.NewWriter(buff)
+	table.SetAutoWrapText(false)
+	table.SetBorder(false)
+	table.SetRowLine(false)
+	table.SetColumnSeparator("")
+	table.Append([]string{m.generateWindowInputView(), m.generateWindowInputHintView()})
```

Comment:
not crucial, but seems input tables location for V3 is slightly shifted (input tables are not in the same column with the result column)

Reply:
updated as suggested, but required a little refactor for more code readability

Reply:
updated as suggested, but required a little refactor for more code readability